sfge/src/main/java/com/salesforce/rules/UnusedMethodRule.java

```diff
@@ -300,6 +483,32 @@ public Boolean visit(ThisMethodCallExpressionVertex vertex) {
         }
     }
 
+    /**
+     * Validator that can determine whether an internal call by a subclass is an invocation of the
+     * targeted method.
+     */
+    private static final class SubclassCallValidator extends CallValidator {
+        private SubclassCallValidator(MethodVertex invokedMethod) {
+            super(invokedMethod);
+        }
+
+        @Override
+        public Boolean visit(MethodCallExpressionVertex vertex) {
+            // TODO: IMPLEMENT THIS METHOD
```

Comment:
When we add support for protected instance methods, that will involve implementing this method.